src/locales/en.json

```diff
@@ -11,7 +11,7 @@
     "precautions": {
       "label": "Precautions",
       "environment_header": "Environment",
-      "environment_question": "Is it indoor or outdoor?",
+      "environment_question": "Is it indoor or outdoor? Ventilation? Air filters?",
```

Comment:
Can we use full sentences to match elsewhere?

"Is it indoor or outdoor? Is there air flow or air filters?

Reply:
cool, makes sense. slight change to keep "ventilation" visible/searchable on page